examples/kubernetes-in-docker/README.md

```diff
@@ -459,26 +448,40 @@ Conjur CLI deployment in the `conjur-oss` namespace:
     $
 ```
 
-The Conjur-CLI pod can be used to execute Conjur commands against the
-Conjur server by first running the following:
+To run
+[Conjur CLI](https://docs.conjur.org/latest/en/Content/Tools/CLI_Help.htm)
+commands, set up a `conjur` command alias in your shell
+that executes commands via the Conjur CLI pod as follows:
 
 ```
-    export CONJUR_ACCOUNT=myConjurAccount
-    export CONJUR_POD="$(kubectl get pod -n conjur-oss -l app=conjur-oss \
-            -o jsonpath='{.items[0].metadata.name}')"
-    export CLI_POD="$(kubectl get pods -n conjur-oss -l app=conjur-cli \
-            -o jsonpath='{.items[0].metadata.name}')"
-    ADMIN_PASSWORD="$(kubectl exec -n conjur-oss $CONJUR_POD -c conjur-oss \
-            -- conjurctl role retrieve-key $CONJUR_ACCOUNT:user:admin | tail -1)"
-    kubectl exec -n conjur-oss $CLI_POD -- conjur authn login -u admin -p $ADMIN_PASSWORD 
-    export CONJUR_CMD="kubectl exec -n conjur-oss $CLI_POD -- conjur"
+# Retrieve Conjur admin password
+CONJUR_POD="$(kubectl get pods -n conjur-oss -l app=conjur-oss \
+        -o jsonpath='{.items[0].metadata.name}')"
+CONJUR_ACCOUNT="$(kubectl exec -n conjur-oss $CONJUR_POD -c conjur-oss \
+        -- printenv \
+        | grep CONJUR_ACCOUNT \
+        | sed 's/.*=//')"
+ADMIN_PASSWORD="$(kubectl exec -n conjur-oss $CONJUR_POD -c conjur-oss \
+        -- conjurctl role retrieve-key $CONJUR_ACCOUNT:user:admin | tail -1)"
+
+# Initialize the Conjur CLI pod's connection to Conjur
+export CLI_POD="$(kubectl get pods -n conjur-oss -l app=conjur-cli \
+        -o jsonpath='{.items[0].metadata.name}')"
+CONJUR_URL="https://conjur-oss.conjur-oss.svc.cluster.local"
+kubectl exec -n conjur-oss $CLI_POD \
+        -- bash -c "yes yes | conjur init -a $CONJUR_ACCOUNT -u $CONJUR_URL"
+kubectl exec -n conjur-oss $CLI_POD -- conjur authn login \
+        -u admin -p $ADMIN_PASSWORD
+
+# Create a 'conjur' command alias
+alias conjur="kubectl exec -n conjur-oss $CLI_POD -- conjur"
```

Comment:
Th alias is a lot cleaner, but the export is a lot more portable. It works in interactive and non-interactive shells, and can be passed to sub-shells. Thoughts ?

Reply:
Why not document both! :) The alias is nice for folks who have shells that support it.
@doodlesbykumbi I've updated the README.me to include both options.

Reply:
I like it 💯